frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -273,11 +287,26 @@ export const ConnectedTopologyView: React.FC<ComponentProps> = ({
           />
         </StackItem>
         <StackItem isFilled className={containerClasses}>
-          <div ref={setViewContainer} className="pf-topology-content">
-            {viewContent}
-            {!model?.nodes?.length ? (
-              <TopologyEmptyState setIsQuickSearchOpen={setIsQuickSearchOpen} />
-            ) : null}
+          <div className="co-file-dropzone co-file-dropzone__flex">
+            <div ref={setViewContainer} className="pf-topology-content">
+              {canDrop && isOver && (
+                <div
+                  className={classNames(
+                    'co-file-dropzone-container',
+                    'co-file-dropzone--drop-over',
+                    'odc-topology__dropzone',
+                  )}
+                >
+                  <span className="co-file-dropzone__drop-text odc-topology__dropzone-text">
+                    {t('topology~Drop file ({{fileTypes}}) here', { fileTypes })}
+                  </span>
+                </div>
+              )}
+              {viewContent}
+              {!model?.nodes?.length ? (
+                <TopologyEmptyState setIsQuickSearchOpen={setIsQuickSearchOpen} />
+              ) : null}
```

Comment:
do we need to check on model as it's the required field? and can have as below

```suggestion
              {!model.nodes?.length && (
                <TopologyEmptyState setIsQuickSearchOpen={setIsQuickSearchOpen} />
              ) }
```